_winners/sidhya-tikku.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Sidhya Tikku
+year: 2018
+org: JBoss Community
+country: 🇮🇳
+links:
+    -
+        icon: globe
```

Comment:
Behance has its own icons, `behance` or `behance-square`, please refer to this site: https://forkaweso.me/Fork-Awesome/icons/